### ISSUE TITLE
Omit explicit "Immune Status" Observations

### ISSUE THREAD

**Issue by jmandel** [dvci/vaccine-credential-ig#4]
This is a murky area; ultimately, it overlaps with the "validation" (risk evaluation) goals that we're explicitly leaving out of scope. So I'd lean toward leaving these kinds of Observations of the the IG for now, since I suspect they may raise concerns in excess of their direct utility.

**Comment by masnick** [comment]
I think it's fine to remove immune status if no one objects.

**Comment by masnick** [comment]
We are going to add in support for lab tests in #44 once work on the vaccination portion of the IG is complete.
